contrib/containerized-pbench/example-workload.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# An example of how to run a simple workload with `pbench-user-benchmark`
+
+# In a containerized environment, the Pbench Agent profile must be run to
+# establish the proper execution environment.
+source /opt/pbench-agent/profile
```

Comment:
If we wanted to...we could set `/opt/pbench-agent/profile` as the entrypoint for the container, so that it is run automagically when the container is invoked.  If we then tweaked that script to, as its last action, run `${@}`, any command line provided to the container invocation would then be run in that environment.

That would remove the need for this step, and it would happen seamlessly and automatically for all uses of the container (unless someone overrode the entrypoint), which seems like a really useful idea, given the purpose of this container.

Thoughts?  (If we didn't want to disturb `/opt/pbench-agent/profile` itself, we could wrap it in another script (a la the `jenkins/runner`) and point the entrypoint at that.)

Reply:
Tracking this suggestion in issue #3310.